ImportStats v1.js

```diff
@@ -182,18 +182,23 @@
     
     jf.parseStatblock = function(statblock) {
 
+        log("Dirty block:\n " + statblock);
         texte = clean(statblock);
+        log("Cleaned block:\n " + texte);
         var keyword = findKeyword(texte);
+        log("Keywords:\n " +  JSON.stringify(keyword, null, 2));
         var section = splitStatblock(texte, keyword);
+        log("Sections:\n " +  JSON.stringify(section, null, 2));
         jf.setCharacter(section.attr.name.trim(), texte.replace(/#/g, '<br>'), section.bio);
         processSection(section);
         return section.attr.name;
     }
 
     function clean(statblock) {
         statblock = unescape(statblock);
+        log("Unescaped block:\n " + statblock);
         statblock = statblock.replace(/–/g, '-');
-        statblock = statblock.replace(/<br[^>]*>/g, '#').replace(/(<([^>]+)>)/ig, "");
+        statblock = statblock.replace(/<p>/g, '#').replace(/(<([^>]+)>)/ig, "");

```

Comment:
This changes from targeting `<br>` tags to `<p>` tags. My observation is that roll20 *always* puts in the `<p>` tags, but doesn't always put in the `<br>`s. This fixes the problem discussed [here](https://app.roll20.net/forum/post/10800206/script-importstats-take-monster-stat-blocks-and-import-them-into-roll20-5e-npc-sheets).

Reply:
Thanks! Can't wait to try it.